packages/ui/src/i18n/dictionaries/authenticator/de.ts

```diff
@@ -66,6 +66,10 @@ export const deDict: AuthenticatorDictionary = {
   'Confirm Sign In': 'Anmeldung bestätigen',
   'Create account': 'Hier registrieren',
   'Enter your password': 'Geben Sie Ihr Passwort ein',
+  'Enter your Password': 'Geben Sie Ihre Passwort',
```

Comment:
Should the corresponding value of `'Enter your Password'` align with the value of `'Enter your password'` (or vice versa)?

Reply:
Updated